src/runtime_src/core/pcie/driver/linux/xocl/subdev/xgq.c

```diff
@@ -2980,6 +2997,45 @@ static int xgq_vmr_remove(struct platform_device *pdev)
 	return 0;
 }
 
+static bool vmr_check_sc_is_ready(struct xocl_xgq_vmr *xgq)
+{
+	struct xgq_cmd_cq_vmr_payload *vmr_status =
+		(struct xgq_cmd_cq_vmr_payload *)&xgq->xgq_cq_payload;
+	int ret = 0;;
+
+	ret = vmr_status_query(xgq->xgq_pdev);
```

Comment:
Maybe it would be better if the following code was used
```suggestion
	int ret = vmr_status_query(xgq->xgq_pdev);
```